tools/main-transpile-wrapper.js

```diff
@@ -0,0 +1,8 @@
+// This file exists because it is the file in the tool that is not automatically
+// transpiled by Babel.
+
+require('source-map-support').install();
+require('babel/register')(require('./babel-config.js')); // #RemoveInProd this line is removed in isopack.js
```

Comment:
Where does Babel store its cache of compiled files? Is there any way to configure that?